documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -0,0 +1,1503 @@
+This may result in more change events to be re-sent after a connector restart.
+Set this parameter to `0` to not send heartbeat messages at all. +
+Disabled by default.
+
+|`heartbeat.topics.prefix`
+|`__debezium-heartbeat`
+|Controls the naming of the topic to which heartbeat messages are sent. +
+The topic is named according to the pattern `<heartbeat.topics.prefix>.<server.name>`.
+
+|`snapshot.delay.ms`
+|
+|An interval in milli-seconds that the connector should wait before taking a snapshot after starting up; +
+Can be used to avoid snapshot interruptions when starting multiple connectors in a cluster, which may cause re-balancing of connectors.
+
+|`snapshot.fetch.size`
+|`2000`
+|Specifies the maximum number of rows that should be read in one go from each table while taking a snapshot.
+The connector will read the table contents in multiple batches of this size. Defaults to 2000.
+
+|`snapshot.lock.timeout.ms`
+|`10000`
+|An integer value that specifies the maximum amount of time (in milliseconds) to wait to obtain table locks when performing a snapshot. If table locks cannot be acquired in this time interval, the snapshot will fail (also see link:#snapshots[snapshots]). +
+When set to `0` the connector will fail immediately when it cannot obtain the lock. Value `-1` indicates infinite waiting.
+
+|`snapshot.select.statement.overrides`
+|
+|Controls which rows from tables will be included in snapshot. +
+This property contains a comma-separated list of fully-qualified tables _(SCHEMA_NAME.TABLE_NAME)_. Select statements for the individual tables are specified in further configuration properties, one for each table, identified by the id `snapshot.select.statement.overrides.[SCHEMA_NAME].[TABLE_NAME]`. The value of those properties is the SELECT statement to use when retrieving data from the specific table during snapshotting. _A possible use case for large append-only tables is setting a specific point where to start (resume) snapshotting, in case a previous snapshotting was interrupted._ +
+*Note*: This setting has impact on snapshots only. Events captured during log reading are not affected by it.
+
+|`source.struct.version`
+|v2
+|Schema version for the `source` block in CDC events; Debezium 0.10 introduced a few breaking +
+changes to the structure of the `source` block in order to unify the exposed structure across
+all the connectors. +
+By setting this option to `v1` the structure used in earlier versions can be produced.
+Note that this setting is not recommended and is planned for removal in a future Debezium version.
+
+|`sanitize.field.names`
+|`true` when connector configuration explicitly specifies the `key.converter` or `value.converter` parameters to use Avro, otherwise defaults to `false`.
+|Whether field names will be sanitized to adhere to Avro naming requirements.
+See xref:configuration/avro.adoc#names[Avro naming] for more details.
+
+|`database.server.timezone`
```

Comment:
Does this option actually exist for this connector?

Reply:
Done